tests/bundle/bundle-test.ts

```diff
@@ -276,9 +273,6 @@ async function runAll() {
   })
   console.info('Credential issued')
 
-  KiltCredentialV1.validateStructure(issued as KiltCredential.Interface)
-  console.info('Credential structure validated')
-
```

Comment:
Not sure if this should be in this PR

Reply:
I chose not to export `KiltCredentialV1` on sdk-js, so this is no longer available in the bundle tests.